woocommerce/woocommerce
# Removed floating semi colon from info type setting control

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:




There was a floating semi-colon on the "info" setting type that didn't look great when using said type.

![image](https://github.com/woocommerce/woocommerce/assets/7070154/43965234-04bd-42a8-b354-c0c3c9a018e4)

Closes #40767 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create an "info" setting type.
2. You'll see a random semi-colon visible in the admin section on a line above the "info" text you would have just added



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 
It's just a small text update

</details>
